src/couch/src/couch_ejson_size.erl

```diff
@@ -21,8 +36,7 @@ ejson_size(List) when is_list(List) ->
     2 + lists:sum([ejson_size(V) + 1 || V <- List]) - 1;
 
 ejson_size(Float) when is_float(Float) ->
-    % this doesn't match what jiffy does. so will be off
-    byte_size(float_to_binary(Float, [{decimals, 16}, compact]));
+    erlang:byte_size(jiffy:encode(Float));
```

Comment:
This is better than before but not ideal in general. It is better because jiffy does a great job at a minimal encoding of floats. Better than Erlang at least. Not ideal because there is very little chance users would be using jiffy to encode their json. (They'd use it indirectly when replicating but they'd add it initially via some other encoder).